extensions/bond/sawtooth_bond/txn_family.py

```diff
@@ -49,6 +50,19 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def _load_config(journal):
+    try:
+        with open('./etc/sawtooth-bond.json') \
+                as bond_fd:
+            bond_config = json.load(bond_fd)
+    except IOError, ex:
+        LOGGER.exception('IOError: %s', str(ex))
+        return
+
+    libor_publisher_addr = bond_config["LiborPublisherAddr"]
```

Comment:
I think you need to guard against the "LiborPublisherAddr" attribute not existing in the configuration file:
libor_publisher_addr = bond_config.get('LiborPublisherAddr')
if libor_publisher_addr is not None:
  ....

This way the code works with the default LIBOR address if one is not provided.  Either that, or flag an error if it doesn't exist.